Xamarin.Forms.Core/Shell/BaseShellItem.cs

```diff
@@ -112,6 +123,51 @@ public bool IsTabStop
 			set => SetValue(IsTabStopProperty, value);
 		}
 
+		public string BadgeText
+		{
+			get { return (string)GetValue(BadgeTextProperty); }
+			set { SetValue(BadgeTextProperty, value); }
+		}
+		public string BadgeMoreText => Shell.GetBadgeMoreText(this);
+
+		public Color BadgeTextColor => Shell.GetBadgeTextColor(this);
+
+		public Color BadgeUnselectedTextColor => Shell.GetBadgeTextColor(this);
+
+		public Color BadgeColor => Shell.GetBadgeColor(this);
+
+		public Color BadgeUnselectedColor => Shell.GetBadgeUnselectedColor(this);
+
+		public Color EffectiveBadgeTextColor => GetEffectiveBadgeTextColor(IsChecked);
```

Comment:
those aren't bindable on the base shell item

Reply:
See answer below.